addons/im_livechat/static/tests/embed/livechat_service_tests.js

```diff
@@ -64,28 +64,30 @@ QUnit.test("Only necessary requests are made when creating a new chat", async (a
     await start({
         mockRPC(route) {
             if (!route.includes("assets")) {
-                assert.step(route);
+                step(route);
             }
             if (route === "/mail/link_preview") {
                 linkPreviewDeferred.resolve();
             }
         },
     });
     await contains(".o-livechat-LivechatButton");
-    assert.verifySteps([
+    await assertSteps([
         "/im_livechat/init",
         "/web/webclient/load_menus", // called because menu_service is loaded in qunit bundle
         "/mail/load_message_failures", // called because mail/core/web is loaded in qunit bundle
     ]);
     await click(".o-livechat-LivechatButton");
-    assert.verifySteps(["/im_livechat/get_session"]);
+    await contains(".o-mail-Message", { text: "Hello, how may I help you?" });
+    await assertSteps(["/im_livechat/get_session"]);
     await insertText(".o-mail-Composer-input", "Hello!");
-    assert.verifySteps([]);
+    await assertSteps([]);
     await triggerHotkey("Enter");
     await contains(".o-mail-Message", { text: "Hello!" });
     await linkPreviewDeferred;
-    assert.verifySteps([
+    await assertSteps([
         "/im_livechat/get_session",
+        "/discuss/channel/fold",
```

Comment:
Note: extra RPC because there was a condition on `store.self` before `init_messaging` that was (incorrectly) failing due to missing data.

Reply:
We might want to remove that RPC at a later time:
- other PR to merge RPC together,
- or if possible better initial fold state so that we don't have to call that at all.